lib/schema.js

```diff
@@ -45,9 +45,8 @@ const schema = Joi.object().keys({
   exemptMilestones: fields.exemptMilestones.default(false),
   staleLabel: fields.staleLabel.default('wontfix'),
   markComment: fields.markComment.default(
-    'This issue has been automatically marked as stale because ' +
-    'it has not had recent activity. It will be closed if no further ' +
-    'activity occurs. Thank you for your contributions.'
+    'Is this still relevant? If so, what is blocking it? ' +
+    'Is there anything you can do to help move it forward?'
```

Comment:
This is a great change, but I'm worried that folks won't understand the repercussion of not commenting. This line feels important:

> It will be closed if no further activity occurs

Could you add that message back in somehow (not the specific wording, but that the issue will be closed if there's no more activity)?